src/common/pony/detail/atomics.h

```diff
@@ -8,10 +8,12 @@
 #  error "Unsupported platform"
 #endif
 
-#ifndef __cplusplus
+ #ifndef __cplusplus
 #  include <stdalign.h>
 #endif
 
+#include <platform.h>
```

Comment:
Nice catch Benoit. We didn't need features.h anymore. I've removed it entirely along with the platform.h inclusion.